nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -118,6 +118,8 @@ Use `configure.packages` instead.
 
 - Add udev rules for the Teensy family of microcontrollers.
 
+- Add derivation for ocs-url with support for ocs:// URI.
```

Comment:
I think news items are written for user-facing changes and major changes. Most packages don't get mentioned in a NEWS file. I would suggest omitting NEWS updates.